CIs fail

Does anyone know why several CIs fail? Did pytest 4.6.0 break some of our tests?

```
_____________ ERROR collecting mne/forward/tests/test_forward.py ______________
<frozen importlib._bootstrap>:900: in _find_spec
    ???
E   AttributeError: 'AssertionRewritingHook' object has no attribute 'find_spec'

During handling of the above exception, another exception occurred:
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:983: in _find_and_load
    ???
<frozen importlib._bootstrap>:963: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:902: in _find_spec
    ???
<frozen importlib._bootstrap>:876: in _find_spec_legacy
    ???
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\_pytest\assertion\rewrite.py:169: in find_module
    source_stat, co = _rewrite_test(self.config, fn_pypath)
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\_pytest\assertion\rewrite.py:404: in _rewrite_test
    rewrite_asserts(tree, fn, config)
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\_pytest\assertion\rewrite.py:453: in rewrite_asserts
    AssertionRewriter(module_path, config).run(mod)
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\_pytest\assertion\rewrite.py:701: in run
    new.extend(self.visit(child))
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\ast.py:262: in visit
    return visitor(node)
c:\hostedtoolcache\windows\python\3.7.3\x64\lib\site-packages\_pytest\assertion\rewrite.py:828: in visit_Assert
    top_condition, explanation = self.visit(assert_.test)
E   TypeError: cannot unpack non-iterable NoneType object
```

--- a/mne/time_frequency/tests/test_tfr.py
+++ b/mne/time_frequency/tests/test_tfr.py
@@ -574,8 +574,9 @@ def test_add_channels():
                for ch in tfr_stim.ch_names + tfr_meg.ch_names)
     tfr_new = tfr_meg.copy().add_channels([tfr_eeg])
 
-    assert all(ch in tfr_new.ch_names
-               for ch in tfr.ch_names if ch != 'STIM 001')
+    have_all = all(ch in tfr_new.ch_names
+                   for ch in tfr.ch_names if ch != 'STIM 001')
+    assert have_all
     assert_array_equal(tfr_new.data, tfr_eeg_meg.data)
     assert all(ch not in tfr_new.ch_names for ch in tfr_stim.ch_names)
 

--- a/mne/viz/tests/test_topomap.py
+++ b/mne/viz/tests/test_topomap.py
@@ -210,9 +210,11 @@ def test_plot_topomap():
     subplot = [x for x in p.get_children() if 'Subplot' in str(type(x))]
     assert len(subplot) >= 1, [type(x) for x in p.get_children()]
     subplot = subplot[0]
-    assert (all('MEG' not in x.get_text()
-                for x in subplot.get_children()
-                if isinstance(x, matplotlib.text.Text)))
+
+    have_all = all('MEG' not in x.get_text()
+                   for x in subplot.get_children()
+                   if isinstance(x, matplotlib.text.Text))
+    assert have_all
 
     # Plot array
     for ch_type in ('mag', 'grad'):

--- a/requirements.txt
+++ b/requirements.txt
@@ -12,7 +12,7 @@ pandas
 numexpr
 python-picard
 statsmodels
-pytest
+pytest!=4.6.0
 pytest-cov
 pytest-faulthandler
 pytest-mock

